tools/istio-iptables/pkg/dependencies/implementation.go

```diff
@@ -71,11 +73,28 @@ type RealDependencies struct {
 }
 
 func (r *RealDependencies) execute(cmd string, ignoreErrors bool, args ...string) error {
-	if r.CNIMode {
```

Comment:
Given the risk here (see linked doc, its not always safe to do this - although I suspect here it is), we may want to add a feature flag to allow disabling this if it causes issues on some platforms, etc

Reply:
ok